core/metrics-core-service/src/main/java/org/hawkular/metrics/core/dropwizard/MetricNameService.java

```diff
@@ -33,28 +33,45 @@
  */
 public class MetricNameService implements MetricFilter {
 
-    public static final String TENANT_ID = "admin";
 
     private static final String separator = ":";
 
-    private String hostname;
+    private final String adminTenant;
+    private final String hostname;
 
     public MetricNameService() {
         try {
             hostname = InetAddress.getLocalHost().getHostName();
         } catch (UnknownHostException e) {
             throw new RuntimeException("Failed to get hostname", e);
         }
+
+        this.adminTenant = "admin";
```

Comment:
Is there any reason why we need to hard code the 'admin' default here when we are already setting the default in ConfigurationKey.java?

Do we need this constructor? or can we get rid of it and rely on the other constructions which require the admin tenant to be passed?